Seneca-CDOT/telescope
# Simplify Search Front-End

## Type of Change




## Description
To be able to finish the [PR for search bar front-end](https://github.com/Seneca-CDOT/telescope/pull/3211), I need to simplify the search. The level of complexity was not required, so I simplified this functionality to be higher up from `SearchPage.tsx` and share across components more uniformly. 


## Steps to test the PR
1. `npm install`
2. `cp config/env.staging .env`
3. `npm run dev`
4. Try using the searchbar - should have the same functionality as previously, meaning nothing should have changed, only the method for submitting front-end changed. 



## Checklist
